src/health/endpoints.rs

```diff
@@ -165,15 +165,20 @@ fn check_peers_connected(state: &ForestState, acc: &mut MessageAccumulator) -> b
 }
 
 fn check_eth_mappings_up_to_date(state: &ForestState, acc: &mut MessageAccumulator) -> bool {
-    match state.settings_store.eth_mapping_up_to_date() {
-        Ok(Some(true)) => {
-            acc.push_ok("eth mappings up to date");
-            true
-        }
-        Ok(None) | Ok(Some(false)) | Err(_) => {
-            acc.push_err("no eth mappings");
-            false
+    if state.config.chain_indexer.enable_indexer {
```

Comment:
Why is `enable_indexer` being tested here? Indexing is specifically designed for mapping Filecoin data in the form `{Cid => bytes}`.
Eth mapping is for `{EthHash => bytes}`.

Reply:
@elmattic `enabel_indexer` is tested in both `maybe_populate_eth_mappings_in_background` and `maybe_start_indexer_service`, is either of them wrongly tested?

Reply:
Ah, sorry you are right. This is what the documentation is saying about `enable_indexer`: 

> Whether or not to index the chain to support the Ethereum RPC API.